standard/expressions.md

```diff
@@ -1275,6 +1278,36 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 > ```
 > Within the `A` class, those occurrences of the Color identifier that reference the Color type are delimited by `**`, and those that reference the Color field are not. *end example*
 
+### §null-conditional-member-access Null Conditional Member Access
+
+A *null_conditional_member_access* consists of a *primary_expression* followed by a "`?.`" token, followed by an *Identifier*, optionally followed by a *type_argument_list*.
+
+```ANTLR
+null_conditional_member_access
+    : primary_expression '?.' Identifier type_argument_list?
+    ;
+```
+
+A  *null_conditional_member_access* expression `E` is of the form `P?.I`. Let `T` be the type of the *member_access*  expression `P.I`. The meaning of `E` is determined as follows:
+
+- If `T` is a type parameter that is not known to be a reference type or a non-nullable value type, a compile-time error occurs.
+- If `T` is a non-nullable value type, then the type of `E` is `T?`, and the meaning of `E` is the same as the meaning of:
+	> ```csharp
+	> ((object)P == null) ? (T?)null : P.I
```

Comment:
Minor note for the future - I'd love to replace the `((object)P == null)` part with `P is null`. We can only do that when pattern matching is implemented, but I think it would be good to do so when we can.

I'm tempted to file a reminder issue... any thoughts?

Reply:
I think it is worth noting regardless of whether we go with PR #7 or this one.